src/app/routes/public/public-crp/public-crp.component.ts

```diff
@@ -28,6 +28,7 @@ import { DialogBoxComponent as ZohoDialogComponent } from '@ws/app/src/lib/route
 // tslint:disable-next-line: import-name
 import _ from 'lodash';
 import { IOrganizationDetails } from './models/public-crp-model';
+import { MobileAppsService } from 'src/app/services/mobile-apps.service';
```

Comment:
please use relative path

Reply:
Please check again changed to relative path